shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -40,10 +41,10 @@ const std::vector<std::string> kBindableSystemKeys = {
 
 namespace flutter {
 
-FlutterTizenView::FlutterTizenView(std::unique_ptr<TizenWindow> window)
-    : window_(std::move(window)) {
-  window_->SetView(this);
-  window_->BindKeys(kBindableSystemKeys);
+FlutterTizenView::FlutterTizenView(std::unique_ptr<TizenViewBase> tizen_view)
+    : tizen_view_(std::move(tizen_view)) {
+  tizen_view_->SetView(this);
+  tizen_view_->BindKeys(kBindableSystemKeys);
```

Comment:
When the type of `tizen_view_` is not window, I don't think this is the role of the Embedder anymore. I think it should be fixed later.

Reply:
Yes, that makes sense. I think we can move `BindKeys` from `TizenViewBase` to `TizenWindow`.

Reply:
I think the view also needs a bind to a special key.
We may also need the TV remote's back key or other special key inside the View.
Since key binding is platform dependent, I think that the information provided for VD profile can be same with either Window or View.

This has already been talked about in private chats.
If necessary, I hope this is discussed in a separate PR regarding special key bindings.

Reply:
Yes, we can handle it in another PR.

Reply:
> I think the view also needs a bind to a special key.
We may also need the TV remote's back key or other special key inside the View.
Since key binding is platform dependent, I think that the information provided for VD profile can be same with either Window or View.

Sorry. I don't still get why it should be the embedder's role to register the special keys. Isn't is possible for the parent app to call `eext_win_keygrab_set` with its own window as necessary?

Reply:
> Sorry. I don't still get why it should be the embedder's role to register the special keys. Isn't is possible for the parent app to call eext_win_keygrab_set with its own window as necessary?

Using eext_win_keygrab_set, the app user can also bind a key to the app window.

my opinion is that the flutter app(window and view) created with flutter-tizen can provide the same key bind within the VD profile. If the user has experienced that BackKey works in flutter-app (win), but does not add it to the app in the view, it may not work. Of course, not binding to the app's window may mean that a special key is not used.
This is just my opinion.

however, it seems problematic for the view to modify the properties of the window.
I can also delete the view's bind code from this PR. Should I do that now?
@bbrto21 told me on chat that he would proceed with the modify related to key bind.